### PR TITLE
Added simpleReflexAgentProgram to vacuumworld.ipynb

### PR DESCRIPTION
**Pull request by Noumanmufc1**
@MrDupin  and @norvig  Please review?

### CODE REVIEW

File: vacuum_world.ipynb [hunk 1]
```diff
@@ -336,21 +474,35 @@
...
+    "            return self.__state == state\n",
+    "        \n",
+    "def interpret_input(state):\n",
+    "    return state\n",
+    "    \n",
+    "loc_A = (0, 0)\n",
+    "loc_B = (1, 0)\n",
     "\n",
-    "rules = None\n",
+    "# create rules for a two state Vacuum Environment\n",
```
Comment (MrDupin) [inline vacuum_world.ipynb]: Add a hyphen ( '-' ) between 'two' and 'state'.

Comment (Noumanmufc1) [reply vacuum_world.ipynb]: Fixed the typo. Review it whenever you are free @MrDupin

**Comment by norvig** [comment]
I'm not a fan of the `Rule` class. I would prefer just having `if` statements.

**Comment by Noumanmufc1** [comment]
@norvig will try to do that with `if` statements

**Pull request closed without merging**